src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -710,7 +710,7 @@ private static void ExcludeFiles(List<PhysicalPackageFile> searchFiles, string b
 
         private static void CreatePart(ZipArchive package, string path, Stream sourceStream)
         {
-            if (PackageHelper.IsNuspec(path) || ProjectJsonPathUtilities.IsProjectConfig(path))
```

Comment:
I just mimicked nuspec behavior in the initial project.json implementation, and slowly found ways in which is was different.